stringcompare/distance/hamming.py

```diff
@@ -0,0 +1,22 @@
+#hamming distance
+from comparator import StringComparator
+
+def hamming(s, t):
+    if (len(s) != len(t)):
```

Comment:
It'd be nice to be able to compare strings of different lengths, there's an increase of 1 in the distance for each difference in length.